tools/scylla-sstable.cc

```diff
@@ -916,7 +916,8 @@ void validate_operation(schema_ptr schema, reader_permit permit, const std::vect
     }
 
     abort_source abort;
-    json_writer writer;
+    std::stringstream ss;
```

Comment:
could you please accompany this with a comment?

also, would be better to rename `ss` to something like `json_output`.